api.js

```diff
@@ -38,16 +24,16 @@ function find(ary, test) {
   return v
 }
 
-module.exports = function (plugins, defaults) {
-
+module.exports = function (plugins, defaultConfig) {
   function create (opts) {
-    opts = merge(merge({}, defaults), opts)
+    opts = merge(merge({}, defaultConfig), opts)
     //change event emitter to something with more rigorous security?
     var api = new EventEmitter()
     create.plugins.forEach(function (plug) {
+      // mix: uhhh where is create.createClient defined?
```

Comment:
hey @dominictarr what is this and where does it come from... I searched files and couldn't find it anywhere

Reply:
oh, looks like that code has been removed. now you use .connect instead (provided by core.js)